tests/scripts/all.sh

```diff
@@ -270,6 +272,11 @@ msg()
     else
         current_section="$1"
     fi
+
+    if [ $QUIET -eq 1 ]; then
+        return
+    fi
+
```

Comment:
Missing: in quiet mode, silence the call to `dd` to create `seedfile`.

Is anything else missing? I haven't tried a complete run.

Reply:
I've actually silenced it unconditionally as I found even without `-q`, its output was more confusing than helpful.

Reply:
Fully suppressing the output means suppressing error messages as well, but that's unavoidable without adding complications in one form or another, so ok. It's annoying that there's other no simple, portable way to obtain N bytes without the garbage output from `dd` (`dd status=none` is a GNU coreutils extension, `head -c` is widespread but not available on e.g. OpenBSD, and I'd prefer not to artificially restrict the portability of `all.sh`).